adele-booking/src/main/java/com/microservicesteam/adele/booking/boundary/web/WebSocketEventPublisher.java

```diff
@@ -16,8 +20,24 @@ public WebSocketEventPublisher(SimpMessagingTemplate template) {
         this.template = template;
     }
 
-    public void publish(Event event) {
+    public void publish(TicketsEvent event) {
         this.template.convertAndSend("/topic/tickets", event);
         log.debug("Event was published: {}", event);
     }
+
+    public void publishToSector(TicketsEvent event) {
+        ImmutableSet<Integer> sectors = extractSectors(event);
+        sectors.forEach(sector -> publishToSector(sector, event));
+    }
+
+    private ImmutableSet<Integer> extractSectors(TicketsEvent event) {
+        return event.positions().stream()
+                .map(Position::sectorId)
+                .collect(toImmutableSet());
```

Comment:
Hmm, user can buy tickets in multiple sectors? I am wondering whether sectorId is in the right place. Shouldn't we move into TicketsEvent instead of Position?

Reply:
Let's deal with this in #59